src/cpu/aarch64/matmul/acl_matmul.hpp

```diff
@@ -109,8 +129,8 @@ struct acl_matmul_t : public primitive_t {
         }
 
         acl_matmul_conf_t amp_;
-
         acl_post_ops_t post_ops;
+        dnnl::impl::format_kind_t weights_format_kind;
```

Comment:
Nit: please make it `weights_format_kind_`.

Reply:
Done!